server/routerlicious/packages/routerlicious/config/config.json

```diff
@@ -262,7 +262,7 @@
         "endpoint": ""
     },
     "worker": {
-        "alfredUrl": "http://alfred:3003",
+        "alfredUrl": "http://alfred:3000",
```

Comment:
Was this port value wrong originally?

Reply:
Looks like it was changed inadvertently in this PR: https://github.com/microsoft/FluidFramework/pull/2217/files

And it wasn't being used anywhere till now except by "gateway", so it wasn't noticed

Reply:
Got it. Thanks for fixing it!